app/src/main/java/pl/szczodrzynski/edziennik/data/api/edziennik/github/Contributor.kt

```diff
@@ -0,0 +1,7 @@
+package pl.szczodrzynski.edziennik.data.api.edziennik.github
+
+class Contributor {
```

Comment:
This should be a data class with a primary constructor. The properties do not need to be mutable.